src/app/home/home.component.html

```diff
@@ -0,0 +1 @@
+<p>home works!</p>
```

Comment:
need to create components folder and all the components you need to add in that folder